docs/product-getting-started-guide/src/main/asciidoc/task_data-model.adoc

```diff
@@ -1,66 +1,157 @@
 [id='_defining_a_data_model']
-= Defining a Data Model
+= Creating the traffic violation data model
 
-The following chapter instructs you how to create a data model in {PRODUCT}.
+The following chapter instructs you how to create a traffic violation data model in {PRODUCT}. Data models are models of data objects. A data object is a custom complex data type (for example, a Violation data object containing data fields Violation Date, Fine Amount, and Speed Limit).
+
+Data models are saved in data models definitions stored in your Project. {PRODUCT} provides the Data Modeler, a custom graphical editor, for defining data objects. *<MH> Ivo, I think that this paragraph might be a bit too low-level for the purpose of this tutorial. If you agree, I will remove it. Please advise</MH>*
```

Comment:
"stored in your project" no cap. Capitalization is something we all need to agree on, but the consensus I believe is lower case almost always, unless direct ref to UI (litereally quoting the UI, as in procedures), or predetermined cases of capitalization (like KIE Server). Here's the [terminology capitalization](https://docs.google.com/spreadsheets/d/1kmK92hY3JJp7gZTvLzd0CawaZzvI2MYYv2uaLaQcMJc/edit#gid=1999150540) decision in progress.

Reply:
oops. fixed.